docs/semantics-of-k.tex

```diff
@@ -4644,6 +4651,49 @@ \subsection{More about Symbols}
 Recall that \KVariableListAsKPatternList takes a list of \KVariableList
 and map it to \KPatternList, defined in Section~\ref{sec:ML-patterns}.
 
+\subsection{Built-ins in the Meta-Theory}
+
+Recall that in Section~\ref{sec:builtins},
+we define matching logic theories \emph{with built-ins} as a tuple
+$(S_\builtin, \Sigma_\builtin, S, \Sigma, A)$,
+where $S_\builtin$ contains built-in sorts
+and $\Sigma_\builtin$ contains built-in symbols.
+Unlike $S$ and $\Sigma$ whose semantics is axiomatized by
+a set $A$ of axioms, the semantics of built-ins are given
```

Comment:
It would be clearer to say "the axiomatization of the semantics of built-ins are given by ...", if that's correct.

Reply:
We don't axiomatize the semantics of built-ins; but I will add some more texts discussing what an oracle is to the document.

Reply:
@bmmoore I've addresse all comments. Please review.